app/views/layouts/angular/_multi_auth_credentials.html.haml

```diff
@@ -261,25 +261,48 @@
       = miq_tab_content('hawkular', 'default') do
         .form-group
           .col-md-12.col-md-12
-            = render :partial => "layouts/angular-bootstrap/endpoints_angular",
-                                 :locals  => {:ng_show                => true,
-                                              :ng_model               => "#{ng_model}",
-                                              :id                     => record.id,
-                                              :ng_reqd_hostname       => "true",
-                                              :ng_reqd_api_port       => "true",
-                                              :prefix                 => "hawkular"}
-            = render :partial => "layouts/angular-bootstrap/auth_credentials_angular_bootstrap",
-                       :locals  => {:ng_show           => true,
-                                    :ng_model          => "#{ng_model}",
-                                    :main_scope        => main_scope,
-                                    :ng_show_userid    => "false",
-                                    :ng_show_password  => "false",
-                                    :ng_show_verify    => "false",
-                                    :validate_url      => validate_url,
-                                    :id                => record.id,
-                                    :prefix            => "hawkular",
-                                    :verify_title_off  => _("hawkular URL and API port fields are needed to perform validation."),
-                                    :basic_info_needed => true}
+            %label.radio-inline.control-label{"for" => "container_hawkular_disabled"}
+              %input{:type         => "radio",
+                     :name         => "event_stream_selection",
+                     :id           => "container_hawkular_disabled",
+                     :ng_checked   => "emsCommonModel.event_stream_selection == hawkular_disabled",
+                     :value        => "hawkular_disabled",
+                     :ng_model     => "emsCommonModel.event_stream_selection",
+                     :ng_change    => "containerEventsChanged()",
+                     :checkchange  => ""}
+              = _("Disabled")
+            %label.radio-inline.control-label{"for" => "container_hawkular_enabled"}
+              %input{:type         => "radio",
+                     :name         => "event_stream_selection",
+                     :id           => "container_hawkular_enabled",
+                     :ng_checked   => "emsCommonModel.event_stream_selection == hawkular_enabled",
+                     :value        => "hawkular_enabled",
+                     :ng_model     => "emsCommonModel.event_stream_selection",
+                     :ng_change    => "containerEventsChanged()",
```

Comment:
We probably don't need this `ng-change` for `container_hawkular_enabled`, since it's business as usual when Hawkular is enabled. Can you test around this a little bit?